src/app/dashboard/flight-table/flight-table.component.scss

```diff
@@ -82,3 +82,8 @@ $cell-padding: $cell-padding-v $cell-padding-h;
     color: $warn-contrast;
   }
 }
+
```

Comment:
what's this change for?

Reply:
I noticed when the table scrolled horizontally, the paginator would also move. Currently it scrolls almost halfway across the container, but if the table were to contain more columns or wider column content, could end up scrolling to a point some  of the buttons could be come inaccessible. This change makes the paginator stay in one place when the table is scrolled horizontally.

Reply:
good demo, makes sense. thanks!